media.css

```diff
@@ -0,0 +1,1373 @@
+    padding-left: 0;
+  }
+
+  .catalog-item__title {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 20px;
+    padding-left: 15px;
+  }
+
+  .catalog-item__description {
+    grid-template-columns: 1.5fr 1fr;
+    padding-left: 15px;
+    padding-right: 7px;
+  }
+
+  .catalog-item__price {
+    font-size: 25px;
+    line-height: 24px;
+  }
+
+  .catalog-item__grey,
+  .catalog-item__black,
+  .catalog-item__red,
+  .catalog-item__orange,
+  .catalog-item__brown,
+  .catalog-item__cherry,
+  .catalog-item__blue {
+    height: 20px;
+    width: 20px;
+  }
+
+  .video-section {
+    height: 756px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    /*padding-bottom: 117px;*/
+    background: linear-gradient(2.6deg, #000 91%, #fff 92%);
+    margin-bottom: 0;
+  }
+
+  .video-section__image {
+    display: flex;
+    justify-content: center;
+    order: 1;
+  }
+
+  .video {
+    display: none;
+  }
+
+  .video-adaptive {
+    display: block;
+    width: 88%;
+    height: 88%;
+    margin: 0 auto;
+  }
+
+  .video-section__content {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    width: 88%;
+    align-items: center;
+  }
+
+  .video-content__h3 {
+    width: 100%;
+    font-size: 30px;
+    line-height: 37px;
+    margin-bottom: 15px;
+    color: #fff;
+    text-transform: uppercase;
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 35px;
+    color: #fff;
+    text-align: center;
+  }
+
+  .stories-content {
+    width: 100%;
+    background-color: #000;
+    color: #fff;
+    margin-top: -5px;
+    margin-bottom: 0;
+    padding-bottom: 138px;
+  }
+
+  .stories-content__h4 {
+    font-size: 30px;
+    line-height: 29px;
+    font-weight: bold;
+    text-align: center;
+    text-transform: uppercase;
+    margin-bottom: 41px;
+    color: #fff;
+  }
+
+  .stories-items {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .stories-items__item {
+    width: 40%;
+  }
+
+  .stories-item__image {
+    display: block;
+    width: 100%;
+    margin: 0 auto;
+    margin-bottom: 33px;
+  }
+
+  .stories-item__title {
+    font-size: 25px;
+    line-height: 24px;
+    font-weight: bold;
+    margin-bottom: 9px;
+  }
+
+  .stories-item__description {
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 39px;
+    min-height: 200px;
+  }
+
+  .stories-item__description-extended {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 0.7fr 0.5fr 1fr;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+    line-height: 14px;
+    font-weight: bold;
+    color: #d1d1d1;
+    opacity: 1;
+    text-align: right;
+  }
+
+  .gallery {
+    display: block;
+    height: 15%;
+    background: linear-gradient(2.6deg, #fff 70%, #000 71%);
+    margin-bottom: -5px;
+    width: 100%;
+    /*padding-bottom: 124px;*/
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)